File: C/Exemplos/1/exec1.c
#include <stdio.h>

int main()
{
    float n1, n2, n3, media;
    printf("Calculo de media de notas\n");
    printf("Digite a primeira nota: \n");
    scanf("%f", &n1);

    printf("Digite a segunda nota: \n");
    scanf("%f", &n2);

    printf("Digite a terceira nota: \n");
    scanf("%f", &n3);

    media = (n1 + n2 + n3) / 3;
    printf("Notas digitadas: %.1f, %.1f, %.1f\n", n1, n2, n3);

    printf("Media do aluno: %.1f", media);

    return 0;
}

File: pascal/Prova/Exec1.pas
Program Pzim ;
type
	TAluno = record
		CPF : string;
		NOME : string;
		NOTAS : array [1..3] of real;
	end;
var 
	aluno : TAluno;
	i:integer;
	media: real;
Begin
  write('Digite o CPF do aluno: ');
  readln(aluno.CPF);
  write('Digite o NOME do aluno: ');
  readln(aluno.NOME);
  for i:=1 to 3 do
  begin
	  write('Digite a NOTA ', i ,' do aluno: ');
	  readln(aluno.NOTAS[i]);
	  media := media + aluno.NOTAS[i];
  end;
  writeln();
  writeln('CPF: ', aluno.CPF);
  writeln('NOME: ', aluno.NOME);
  writeln('MEDIA: ', media/3:0:2);
  
End.